libs/math/tests/unit/vectors/vector_tests.cpp

```diff
@@ -208,6 +208,7 @@ TYPED_TEST(VectorRegisterTest, minmax_tests)
     b[i] = fetch::math::Type<type>(
         std::to_string((static_cast<double>(std::rand()) / static_cast<double>(RAND_MAX)) *
                        static_cast<double>(fetch::math::numeric_max<type>()) / 2.0));
+    b[i]     = b[i] == 0 ? a[i] : b[i];
```

Comment:
is this just to make sure you don't divide by 0 later?

Reply:
yes